fitbenchmarking/controllers/scipy_go_controller.py

```diff
@@ -79,11 +79,10 @@ def fit(self):
         fun = self.cost_func.eval_cost
         bounds = self.value_ranges
         algorithm = getattr(optimize, self.minimizer)
-        result = algorithm(fun, bounds, **kwargs)
-        self._popt = result.x
-        if result.success:
+        self._result = algorithm(fun, bounds, **kwargs)
+        if self._result.success:
```

Comment:
There is a possibility to clean up the code a little bit here as well. Now that you have made `results` a class variable. You can do this is in **cleanup** method and remove `self._status` from the class.

```
if self._result.success:
      self.flag = 0
elif "Maximum number of iteration" in self._result.message:
      self.flag = 1
else:
       self.flag = 2
```

Reply:
Good point - will make this change!